Cargo.toml

```
@@ -13,7 +13,7 @@ futures = { version = "0.3", default-features = false }
 futures-timer = { version = "3.0", optional = true }
 log = { version = "0.4", optional = true }
 num = { package = "num-traits", version = "0.2", default-features = false }
-parity-scale-codec = { version = "2.0", default-features = false, features = ["derive"], optional = true }
+parity-scale-codec = { version = "2.2.0-rc.1", default-features = false, features = ["derive"], optional = true }
```

```suggestion
parity-scale-codec = { version = "2.2.0-rc.2", default-features = false, features = ["derive"], optional = true }
```

No point depending on the old macro anywhere, even though it won't really make a difference in this case.